src/azul/health.py

```diff
@@ -251,9 +251,9 @@ def update_cache(self) -> None:
         self.storage_service.put(object_key=f'health/{self.lambda_name}',
                                  data=json.dumps(health_object).encode())
 
-    all_properties: Set[health_property] = {
+    all_properties: Tuple[health_property] = tuple(
         p for p in locals().values() if isinstance(p, health_property)
```

Comment:
This means that the ordering of the result depends on the ordering of the entries in the dictionary returned by locals() which I assume is in the order with which the symbols are defined in the module. While it is deterministic, I think that's a bit arbitrary. I also don't like losing the uniqueness guarantees that come with `Set`. And I don't like how this now mixes `Tuple` for `all_properties` and `Set` for `all_keys`. If we have to use a sequence, which I hope we don't, I would prefer `list` over `tuple`.

TL;DR I think we should address the non-determinism at a higher level by sorting explicitly in app.py.

Reply:
Which was your original approach, anyways, I realize now.

Reply:
My bad for the unfavorable suggestion. I'm confused why `list` is preferred to `tuple` if it's not something you plan on mutating, but I suppose that's besides the point.